packages/mookme/src/commands/init.ts

```diff
@@ -12,6 +12,7 @@ export function addInit(program: commander.Command): void {
     .option('--added-behaviour <added-behaviour>', 'Provide added behaviour and skip the associated prompter')
     .option('--skip-types-selection', 'Skip hook types selection')
     .option('--yes', 'Skip confirmation prompter')
+    .option('-t, --type <type>', 'A valid git hook type ("pre-commit", "prepare-commit", "commit-msg", "post-commit")')
```

Comment:
I would prefer that it is a `--types` options, so that we can select multiple (but still, not all) hooks types

See https://github.com/tj/commander.js#variadic-option

I will probably deprecate `--skip-types-selection` in favor of a more explicit `--all-types` option at some point after your upgrade lands in the main branch :)